meilisearch-rails.gemspec

```diff
@@ -79,19 +53,18 @@ Gem::Specification.new do |s|
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
       s.add_runtime_dependency(%q<json>, [">= 1.5.1"])
-      s.add_runtime_dependency(%q<meilisearch>)
+      s.add_runtime_dependency(%q<meilisearch>, [">= 0.15.2"])
```

Comment:
Is it mandatory to give the version?

Reply:
The second argument corresponds to the requirements of the given gem, so I'd say it's better if we keep it? https://docs.ruby-lang.org/en/2.6.0/Gem/Specification.html#method-i-add_runtime_dependency

Reply:
I'm afraid we use an obsolete version of the gem in our tests... the best would have the version of the one used by meilisearch-rails.

Reply:
I would rather have the last version than an obsolete version running in the tests.

Reply:
Oh, so should I update it to [">= 0.15.**3**"]? Is that what you mean?

Reply:
Checked with @CaroFG, it's ok for me